Bonsai.Tinkerforge/GPSTimerTest.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Tinkerforge;
+
+namespace Bonsai.Tinkerforge
+{
+    [DefaultProperty(nameof(Uid))]
+    [Description("Debug Timer from a GPS Bricklet 2.0.")]
+    public class GPSTimerTest : Combinator<IPConnection, DateTimeData>
```

Comment:
If we are able to split the data streams of GPSV2 into separate operators as suggested above, then this operator might not be necessary (or could be renamed to something like GPSV2DateTime or something like that).